src/main/java/com/example/demo/Controllers/MemberController.java

```diff
@@ -0,0 +1,39 @@
+package com.example.demo.Controllers;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import com.example.demo.Models.Member;
+import com.example.demo.Repositories.MemberRepository;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class MemberController {
```

Comment:
If you take a look at all your Request defined in this controller you can see that all paths have something in common:
` /api/members`
Is there a way to define the path for the Resource only once and then adding what you need to add?
If you find a way to do that your mappings in the methods should be something like:
```
 @PostMapping
   ...
 @DeleteMapping("/{id}")
```

Reply:
The @RequestMapping annotation would help mapping a specific request path or pattern onto my controller